pkg/controller/operators/operator_controller_test.go

```diff
@@ -92,7 +140,6 @@ var _ = Describe("Operator Controller", func() {
 
 			AfterEach(func() {
 				for _, obj := range objs {
-					Expect(k8sClient.Get(ctx, testobj.NamespacedName(obj), obj.(client.Object))).To(Succeed())
```

Comment:
Was this removed because the delete call would fail anyways?

Reply:
As I understood this, this was removed because it's unnecessary to `Get` the object before deleting it. It seems like the goal of this `AfterEach` block is to ensure the objects are deleted.